src/libasr/asr_utils.h

```diff
@@ -698,8 +698,7 @@ static inline ASR::symbol_t *get_asr_owner(const ASR::expr_t *expr) {
             return ASRUtils::get_asr_owner(ASR::down_cast<ASR::FunctionCall_t>(expr)->m_name);
         }
         default: {
-            throw LCompilersException("Cannot find the ASR owner of underlying symbol of expression "
-                                        + std::to_string(expr->type));
+            return nullptr;
```

Comment:
Note: this changes the API, now it will return `nullptr` if the node does not have an ASR owner.